rts/motoko-rts-tests/src/gc/utils.rs

```diff
@@ -28,3 +31,15 @@ pub fn read_word(heap: &[u8], offset: usize) -> u32 {
 pub fn write_word(heap: &mut [u8], offset: usize, word: u32) {
     (&mut heap[offset..]).write_u32::<LE>(word).unwrap()
 }
+
+/// Make a scalar value to be used in heap object payload
+pub fn make_scalar(value: u32) -> u32 {
+    // Scalar values can be at most 31 bits
+    assert_eq!(value >> 31, 0);
```

Comment:
does the assert disappear in production?

Reply:
No, that would be `debug_assert_eq`. But even if I used that, this is in tests, and tests are run in debug mode (which does more checking, for example overflows in arithmetic is checked in debug mode).

Reply:
Nvm.